sys/net/routing/rpl/rpl_control_messages.c

```diff
@@ -539,19 +532,6 @@ void rpl_recv_DIO(void)
             return;
         }
     }
-    else if (my_inst == NULL) {
-        DEBUGF("Not joined an instance yet\n");
-    }
```

Comment:
I dont see the importance of this debug message. If the rpl implementation did not join any instance yet, it will do it now with the information specified in this incoming DIO